crates/flox-rust-sdk/src/models/floxmeta/environment.rs

```diff
@@ -397,3 +454,178 @@ pub enum ManifestError<Git: GitProvider> {
     #[error("Failed parsing 'manifest.json': {0}")]
     ParseManifest(serde_json::Error),
 }
+
+#[derive(Error, Debug)]
+pub enum DeleteSymlinksError<Git: GitProvider> {
+    #[error(transparent)]
+    Metadata(#[from] MetadataError<Git>),
+    #[error("Found generation with last active timestamp in the future: {0}")]
+    TimestampInFuture(String),
+}
+
+#[cfg(test)]
+mod tests {
+
+    use std::marker::PhantomData;
+
+    use chrono::Days;
+
+    use super::*;
+    use crate::flox::Flox;
+    use crate::models::floxmeta::floxmeta_tests::flox_instance;
```

Comment:
There's a few `flox_instance` functions - should we factor them all out? If so where?